### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by miguelpayet**
Hi, I'm going through the book and will post PR's for any typos that I find.

### CODE REVIEW

File: manuscript/chapter1.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Because of this reason, Facebook introduced *create-react-app* as a zero-configu
 
 ## Zero-Configuration Setup
 
-In the Road to learn React you will use [create-react-app](https://github.com/facebookincubator/create-react-app) to bootstrap your application. It's an opinionated yet zero-configuration starter kit for React introduced by Facebook in 2016. People would [recommend it to beginners by 96%](https://twitter.com/dan_abramov/status/806985854099062785). In *create-react-app* the tooling and configuration evolve in the background while the focus is on the application implementation.
+In the road to learn React you will use [create-react-app](https://github.com/facebookincubator/create-react-app) to bootstrap your application. It's an opinionated yet zero-configuration starter kit for React introduced by Facebook in 2016. People would [recommend it to beginners by 96%](https://twitter.com/dan_abramov/status/806985854099062785). In *create-react-app* the tooling and configuration evolve in the background while the focus is on the application implementation.
```
Comment (nicoder) [inline manuscript/chapter1.md:151]: I think "Road" here is correctly capitalised since it is the title of the book ?

Comment (rwieruch) [reply manuscript/chapter1.md:151]: That's right. Thanks for spotting it @nicoder But all the remaining changes are great. Thankg you @miguelpayet 👍

Comment (rwieruch) [reply manuscript/chapter1.md:151]: I will merge it for now and revert Road to be capitalised :)

**Comment by miguelpayet** [comment]
I agree, did not realize you were mentioning the book title. You could consider using italics for this:  http://rasmussen.libanswers.com/faq/32501 and: https://www.grammarly.com/blog/underline-or-italicize-book-titles/

**Pull request merged**